keyboards/earth_rover/readme.md

```diff
@@ -0,0 +1,22 @@
+# earth_rover
+
+![earth_rover](https://pbs.twimg.com/media/Eqh29nYUUAA-9NE?format=jpg)
+
+*Mechanical warm, or just cubic macro pad*
```

Comment:
The italics are just to call attention to this line before it's filled out; the line isn't intended to be kept italicized.

```suggestion
Mechanical warm, or just cubic macro pad.
```